src/components/DropDownMenu.js

```diff
@@ -15,7 +18,7 @@ const DropDownMenu = (props) => {
        </Link>
        <div className="navbar-dropdown is-hidden-mobile is-boxed">
         {keys &&( keys.map(( message ) => (
-         <a className="navbar-item" href="#">
+         <a className="navbar-item" href={"/" + props.langKey + "/" + menu[message][sel] + "/"}>
```

Comment:
could be interesting move the link creation outside the component, in fact in case of external links this is unusable.